Sources/App/Models/Package.swift

```diff
@@ -140,6 +142,36 @@ extension Package {
             }
         })
     }
+
+    func versionUrl(for reference: Reference) -> String {
+        switch (hostingProvider, reference) {
+            case let (.github, .tag(_, tagName)):
+                return "\(url)/releases/tag/\(tagName)"
+            case let (.github, .branch(branchName)):
+                return "\(url)/tree/\(branchName)"
+            case let (.gitlab, .tag(_, tagName)):
+                return "\(url)/-/tags/\(tagName)"
+            case let (.gitlab, .branch(branchName)):
+                return "\(url)/-/tree/\(branchName)"
+        }
```

Comment:
This is more for illustration for if/when we add other providers. I wanted to avoid hard-coding the release urls in the RSS feed itself. The Gitlab ones are the actual urls it uses.